src/utils/tableDataUtils.ts

```diff
@@ -0,0 +1,88 @@
+import { getOffset } from '@/helpers'
+import { TableDataParams } from './tableDataUtils.types'
+
+function sortEntities(
+  items: {
+    name: string
+    mesh: string
+  }[],
+) {
+  return [...items].sort((a, b) => (a.name > b.name ? 1 : a.name === b.name ? (a.mesh > b.mesh ? 1 : -1) : -1))
+}
+
+const getItems = (response: {
+  total: number
+  items: {
+    name: string
+    mesh: string
+  }[]
+}) => {
+  const r = response
```

Comment:
What is the purpose of this assignment?

Reply:
None, just copy-paste of function from which it was taken. Fixed.